tests/unit/mainwindow/test_tabwidget.py

```diff
@@ -38,6 +38,7 @@ def widget(self, qtbot, monkeypatch, config_stub):
         qtbot.addWidget(w)
         monkeypatch.setattr(tabwidget.objects, 'backend',
                             usertypes.Backend.QtWebKit)
+        monkeypatch.setattr(w.tabBar(), 'width', w.width)
```

Comment:
I wonder if we shouldn't just do `w.show()` here, which causes Qt to correctly calculate the sizes.